Assets/Mirror/Components/NetworkAnimator.cs

```diff
@@ -122,7 +132,7 @@ bool CheckAnimStateChanged(out int stateHash, out float normalizedTime, int laye
                     animationHash[layerId] = 0;
                     return true;
                 }
-                return false;
+                return false || change;
```

Comment:
I think these can simplified to just changed
```suggestion
                return change;
```